app/Http/Controllers/TestController.php

```diff
@@ -0,0 +1,11 @@
+<?php
+namespace App\Http\Controllers;
+
+class TestController extends Controller{
+	public function test(){
+		?:?:
+		return 1;
+		return 2;
+		return 3;
+	}
+}
```

Comment:
__[code_sniffer]__
* Expected 1 newline at end of file; 0 found

<!-- Comment by [SideCI](https://sideci.com) -->